package.json

```diff
@@ -76,6 +76,7 @@
     "gh-pages": "^0.11.0",
     "html": "0.0.10",
     "iflow-react-router": "^1.1.17",
+    "ignore-styles": "^5.0.1",
```

Comment:
This is required for running code in node where css modules are required (we now import symbols from `app.js`)